utility/create_views.sql

```diff
@@ -1505,6 +1623,63 @@ CREATE OR REPLACE VIEW ltj_avoin.metsaverkosto AS
         END AS kohdeteksti,
     'https://kartta.hel.fi/paikkatietohakemisto/metadata/?id=322&l=fi'::text AS metadata,
     ('https://kartta.hel.fi/applications/ltj/reports/kohderaportti.aspx?id='::text || kohde.id) AS kohderaportti
-   FROM (public.kohde
-     JOIN public.luokka ON (((luokka.tunnus)::text = (kohde.luokkatunnus)::text)))
+   FROM (ltj.kohde
+     JOIN ltj.luokka ON (((luokka.tunnus)::text = (kohde.luokkatunnus)::text)))
   WHERE (((kohde.luokkatunnus)::text = 'MVER'::text) AND (kohde.voimassa = true) AND (kohde.suojaustasoid = 3));
+  
+
+-- ltj_kohteet-view for SpatialWeb
+
+CREATE OR REPLACE VIEW ltj.ltj_kohteet AS
+SELECT kohde.id,
+    st_geometryn(st_force2d(kohde.geometry1), 1)::geometry(Geometry,3879) AS geometry1,
+    kohde.tunnus,
+    kohde.luokkatunnus,
+    kohde.nimi,
+    kohde.kuvaus,
+    kohde.huom,
+    kohde.voimassa,
+    kohde.numero,
+    kohde.pinta_ala,
+    kohde.suojaustasoid AS suojaustasokohde,
+    kohde.teksti AS tieto,
+        CASE
+            WHEN NOT kohde.teksti_www::text = ''::text THEN kohde.teksti_www
+            ELSE kohde.teksti
+        END AS tieto_www,
+    array_to_string(array_agg(arvo.luokka), ','::text) AS suojelu_arvo_luokka,
+    array_to_string(array_agg(arvo.selite), ', '::text) AS suojelu_arvo_selite
+   FROM ltj.kohde
+     LEFT JOIN ltj.arvo_kohde ON arvo_kohde.kohdeid = kohde.id
+     LEFT JOIN ltj.arvo ON arvo.id = arvo_kohde.arvoid
+  WHERE kohde.voimassa AND st_isvalid(kohde.geometry1)
+  GROUP BY kohde.id;
+
+  
+-- ltj_lajikohteet-view for SpatialWeb
+  
+CREATE OR REPLACE VIEW ltj.ltj_lajikohteet AS
+ SELECT kohde.id,
+    kohde.luokkatunnus,
+    kohde.tunnus,
+    kohde.nimi,
+    (public.st_geometryn(public.st_force2d(kohde.geometry1), 1))::public.geometry(Geometry,3879) AS geometry1,
+    kohde.suojaustasoid AS suojaustasokohde,
+    lajihavainto.suojaustasoid AS suojaustasohavainto,
+    lajirekisteri.suojaustasoid AS suojaustasolaji,
+    luokka.nimi AS luokka,
+    lajihavainto.lajid
+   FROM (((ltj.kohde
+     JOIN ltj.luokka ON (((luokka.tunnus)::text = (kohde.luokkatunnus)::text)))
+     JOIN ltj.lajihavainto ON ((kohde.id = lajihavainto.kohdeid)))
+     JOIN ltj.lajirekisteri ON ((lajihavainto.lajid = lajirekisteri.id)))
+  WHERE (kohde.voimassa AND public.st_isvalid(kohde.geometry1));
+
+
+  -- New table for service metadata
+  
+ CREATE TABLE jakelumetadata (
+    id integer NOT NULL,
+    datanomistaja character varying(25),
+    paivitetty_tietopalveluun date,
```

Comment:
Comma at the end of line causes a syntax error. The last line must be without comma.